### DISCUSSION TITLE
[RFC Call] Agenda - January 25, 2021

### DISCUSSION THREAD

**Discussion by jonathantneal** [withastro/roadmap#88] [Proposal]
# Agenda

- @benjaminwy - collect feedback - https://github.com/withastro/rfcs/discussions/80
- @zadeviggers - collect feedback - https://github.com/withastro/rfcs/discussions/79
- @boehs - collect feedback - https://github.com/withastro/rfcs/discussions/84

--- 

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)_

**Comment by jonathantneal** [comment]
We have several user contributions that we could review for early feedback. We should pick 2 or 3 from this list.

- https://github.com/withastro/rfcs/discussions/79
- https://github.com/withastro/rfcs/discussions/80
- https://github.com/withastro/rfcs/discussions/82
- https://github.com/withastro/rfcs/discussions/83
- https://github.com/withastro/rfcs/discussions/84
- https://github.com/withastro/rfcs/discussions/85
- https://github.com/withastro/rfcs/discussions/87

I’ll review them later to make my own selections, unless popular demand happens first.

**Reply by okikio** [reply]
I like #80, it would be genuinely useful for development and testing

**Reply by zadeviggers** [reply]
I'd love it if #79 got discussed!

**Comment by jonathantneal** [comment]
Notes from today’s call (2022-01-25) can be found within each issue.
